module_tunnels/koupleless_http_tunnel/http_tunnel.go

```
@@ -226,24 +227,32 @@
+			logger.Info("install biz success: ", data.BizName, data.BizVersion)
+			httpTunnel.onOneBizDataArrived(nodeName, vkModel.BizStatusData{
+				Key:        utils.GetBizIdentity(data.BizName, data.BizVersion),
+				Name:       data.BizName,
+				State:      string(vkModel.BizStateActivated),
+				ChangeTime: time.Now(),
+				Reason:     fmt.Sprintf("%s:%s %s succeed", data.BizName, data.BizVersion, data.Command),
+				Message:    data.Response.Data.Message,
+			})
 			return
+		} else {
+			logger.Error(nil, fmt.Sprintf("biz operation failed: %s\n", data.Response))
```

golangci-lint (1.62.2)

printf: fmt.Sprintf format %s has arg data.Response of wrong type github.com/koupleless/module_controller/module_tunnels/koupleless_http_tunnel/ark_service.ArkResponse[github.com/koupleless/arkctl/v1/service/ark.ArkResponseData]

(govet)